src/index.ts

```diff
@@ -555,7 +555,7 @@ export const plugin: PluginFunction<TypescriptMocksPluginConfig> = (schema, docu
     const enumValuesConvention = config.enumValues || 'change-case-all#pascalCase';
     const typeNamesConvention = config.typeNames || 'change-case-all#pascalCase';
     const transformUnderscore = config.transformUnderscore ?? true;
-    const listElementCount = config.listElementCount > 0 ? config.listElementCount : 1;
+    const listElementCount = Math.max(0, config.listElementCount ?? 1);
```

Comment:
Why do you need to use `Math.max`?
Without it it does the same result
![CleanShot 2023-04-19 at 10 54 31](https://user-images.githubusercontent.com/1867939/233115298-3c0cbd7e-f244-4f31-a1d8-e72e1b9931da.png)
![CleanShot 2023-04-19 at 10 54 07](https://user-images.githubusercontent.com/1867939/233115312-fdb86142-cbe8-4d56-9ad0-a4c52d547546.png)

Reply:
I was thinking about preventing negative values in this case, which the old code did. Does it make sense?

Reply:
ok sure